src/components/AppLayout/Content.tsx

```diff
@@ -6,12 +6,25 @@ import { borderHeight, savebarHeight, topBarHeight } from "./consts";
 interface ContentProps {
   [key: `data-${string}`]: string;
   children: React.ReactNode;
+  noSavebar?: boolean;
+  noTopNav?: boolean;
```

Comment:
suggestion: why not `hasSidebar` and `hasTopNav` ?

Reply:
I was wondering about this - but there are only 3 places now which has `noSavebar` - but in the next PR I want to do some more refactoring and make it it's own component instead of prop